apps/emqx_bridge_kafka/src/emqx_bridge_kafka.erl

```diff
@@ -414,6 +415,14 @@ fields(producer_kafka_opts) ->
                     desc => ?DESC(partition_count_refresh_interval)
                 }
             )},
+        {partitions_limit,
+            mk(
+                hoconsc:union([all_partitions, pos_integer()]),
+                #{
+                    default => <<"all_partitions">>,
```

Comment:
I thought that the default value for atoms should be atoms.  Is it indifferent in this case?